webapi/src/main/scala/org/knora/webapi/routing/RouteUtilV1.scala

```diff
@@ -273,4 +273,34 @@ object RouteUtilV1 {
     } else {
       throw BadRequestException(s"MIME type ${fileMetadataResponse.internalMimeType} not supported in Knora API v1")
     }
+
+  def verifyNumberOfParams[A](
```

Comment:
Where are this and the following functions coming from..?

Reply:
I have extracted duplicate code in the Route itself in order to make it easily reusable within the other routes.